lntest/itest/log_error_whitelist.txt

```diff
@@ -153,14 +153,24 @@
 <time> [ERR] RPCS: [/lnrpc.Lightning/SendPayment]: rpc server shutting down
 <time> [ERR] RPCS: [/lnrpc.Lightning/SendToRoute]: rpc error: code = Canceled desc = context canceled
 <time> [ERR] RPCS: [/lnrpc.Lightning/SendToRoute]: rpc error: code = DeadlineExceeded desc = context deadline exceeded
+<time> [ERR] RPCS: [/lnrpc.Lightning/SendToRoute]: context canceled
+<time> [ERR] RPCS: [/lnrpc.Lightning/SendToRoute]: context deadline exceeded
 <time> [ERR] RPCS: [/lnrpc.Lightning/SendToRoute]: rpc server shutting down
 <time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelEvents]: rpc error: code = Canceled desc = context canceled
-<time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: ChannelRouter shutting down
-<time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: router not started
+<time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelEvents]: rpc error: code = DeadlineExceeded desc = context deadline exceeded
+<time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelEvents]: context canceled
+<time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelEvents]: context deadline exceeded
 <time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: rpc error: code = Canceled desc = context canceled
 <time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: rpc error: code = DeadlineExceeded desc = context deadline exceeded
+<time> [ERR] RPCS: [/lnrpc.Lightning/SubscribeChannelGraph]: context canceled
```

Comment:
This makes me think that perhaps we should add a new regular expression in `log_substitutions.txt` that replaces the `/lnrpc.Lightning/xxx` with just `<URL>` then we could collapse all of these context errors into one line in the whitelist.